Merge dockerized rsyslog. config split into multiple files.

t0ffel opened the pull request:
Please wait for @portante to check this before merging.

@portante everything here as discussed today.

I didn't fix ipaddr4.
pid - is very suspicious. a lot of cases need to be checked.

t0ffel committed:
Rename bitscout to viaq
use Peter's copr repository for rsyslog 8.17,
Use external config file outside of the container

t0ffel committed:
Split normalizer rsyslog in separate configs
Apply new data model v2016.03.10.0

t0ffel committed:
fixed rsyslog config following discussion w/ peter

portante commented on Dockerfile:
```diff
@@ -1,18 +1,19 @@
-RUN yum install -y rsyslog rsyslog-elasticsearch rsyslog-gssapi \
+RUN curl https://copr.fedorainfracloud.org/coprs/portante/rsyslog-8.17/repo/epel-7/portante-rsyslog-8.17-epel-7.repo > /etc/yum.repos.d/portante-rsyslog-v8.17-epel-7.repo && \
+    yum install -y rsyslog rsyslog-elasticsearch rsyslog-gssapi \
     rsyslog-mmjsonparse rsyslog-mmsnmptrapd && \
```
I think we can drop the `mmsnmptrapd` module here.

t0ffel committed:
removed installing rsyslog-mmsnmptrapd.
Added 4 worker threads in the main queue.
fixed typo in the config,
default to non-debug rsyslog

t0ffel commented:
@portante please check

richm commented:
looks good to me - @portante ?
